src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleDocumentKey);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleDocumentKey);
+  }
+
+  handleDocumentKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {pressedKey
+            ? `The last pressed key is ${pressedKey}`
+            : 'Nothing was pressed yet'}
+        </p>
```

Comment:
Our letters on page must be between [ ] as in the example

Reply:
That`s why I didn't pass the tests. Thanks ❤️